api/model.go

```diff
@@ -124,6 +124,11 @@ type SnapshotCR struct {
 type BackupTarget struct {
 	client.Resource
 	engineapi.BackupTarget
+	SyncBackupTarget bool `json:"syncBackupTarget"`
+}
+
+type SyncBackupTarget struct {
+	SyncBackupTarget bool `json:"syncBackupTarget"`
```

Comment:
It's weird to have the same name as the struct, or we already did this pattern in the current code?

Reply:
Moved to struct SyncBackupResource.
Thanks.